mne/tests/test_evoked.py

```diff
@@ -34,22 +34,32 @@
 def test_decim():
     """Test evoked decimation."""
     rng = np.random.RandomState(0)
-    n_epochs, n_channels, n_times = 5, 10, 20
+    n_channels, n_times = 10, 20
     dec_1, dec_2 = 2, 3
     decim = dec_1 * dec_2
-    sfreq = 1000.
+    sfreq = 10.
     sfreq_new = sfreq / decim
-    data = rng.randn(n_epochs, n_channels, n_times)
-    events = np.array([np.arange(n_epochs), [0] * n_epochs, [1] * n_epochs]).T
+    data = rng.randn(n_channels, n_times)
     info = create_info(n_channels, sfreq, 'eeg')
     info['lowpass'] = sfreq_new / float(decim)
-    epochs = EpochsArray(data, info, events)
```

Comment:
why did you remove this test?

Reply:
Because it was testing the decimation of epochs, not evoked. I thought it was a copy/paste error

Reply:
did you check if it's tested elsewhere?

Reply:
yes, an exact copy of these tests exists in `test_epochs.py`.